src/cdk/coercion/string-array.spec.ts

```diff
@@ -0,0 +1,28 @@
+import {coerceStringArray} from '@angular/cdk/coercion/string-array';
+
+describe('coerceStringArray', () => {
+    it('should split a string', () => {
```

Comment:
The indentation in this change is off; it should be +2 (and +4 for wrapped lines)